src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] As requested by the previous code reviewer, kindly change your `App.js` component to a `class-based` component as it is a requirement for this project.